src/synchronizer.ts

```diff
@@ -39,18 +40,65 @@ export class Synchronizer {
     }
   }
 
-  async updateAllIssues(excludeClosed: boolean = false): Promise<boolean> {
+  /**
+   * Gets the field node data ids to set custom fields
+   * This method can fail when it doesn't find the correct information so, instead of throwing, we log the error
+   * @param project Project node data. Should be obtained from project kit
+   * @param customField key value pair with the names of the fields. Not case sensitive
+   * @returns Returns a key value pair with the node id of both the field and the value or null if the application threw an error
+   */
+  private async getCustomFieldNodeData(project: NodeData, customField: FieldValues): Promise<FieldValues | null> {
+    try {
+      return await this.projectKit.fetchProjectFieldNodeValues(project, customField);
+    } catch (e) {
+      this.logger.notice("Failed fetching project values. Skipping project field assignment.");
+      this.logger.warning(e as Error);
+      return null;
+    }
+  }
```

Comment:
I made this so, in the case that a project changes a field name or has the wrong field name written, instead of failing it will inform the user in a warning.

It will still assign the issue to the project, it will just not assign the state.

What do you think @paritytech/opstooling ? Should I keep it like this or make it crash instead?

Reply:
My preference would be to keep it like this - assign the issue to the project, not assign the state.

Reply:
My gut is - if this is something that user needs to pay attention to keep it working, then it should fail loudly, not just notice
The project field could be changed by someone without any clue where it would fail, because there's no obvious connection b/w how field is named in a project and the automations set up. So if the field name is "hardcoded" somewhere in configs, and it's been changed by someone, i'd expect to be notified that something just stopped working so I could followup with config fix or rollback the renaming

Reply:
Ok, I will change it to fail then. That should be quite loud.

I only need to do a small change where I change the order of the elements